<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WyszukiwarkaWidget</class>
 <widget class="QWidget" name="WyszukiwarkaWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="search_criteria_groupbox">
     <property name="title">
      <string>Kryteria wyszukiwania</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="layer_label">
        <property name="text">
         <string>Wybierz warstwę:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="layer_combobox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="attribute_label">
        <property name="text">
         <string>Wybierz atrybut:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="attribute_combobox"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="value_label">
        <property name="text">
         <string>Wpisz wartość:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="value_lineedit"/>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="exact_match_checkbox">
          <property name="toolTip">
           <string>Zaznacz, aby szukać wartości, która jest identyczna z wpisanym tekstem.</string>
          </property>
          <property name="text">
           <string>Szukaj dokładnego dopasowania</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="search_button">
          <property name="text">
           <string>Szukaj</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="results_groupbox">
     <property name="title">
      <string>Wyniki</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableWidget" name="results_table"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="select_all_button">
          <property name="text">
           <string>Zaznacz wszystkie z listy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="select_selected_button">
          <property name="text">
           <string>Zaznacz wybrane z listy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="zoom_button">
          <property name="text">
           <string>Przybliż do obiektu</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="output_widget_placeholder" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
